src/utils/string/index.ts

```diff
@@ -162,6 +162,14 @@ export const sortFunctionDate = (a, b, key) => {
  */
 export const applyStrictRegex = (str: string) => `^${str}$`;
 
+/**
+ *
+ * @param str - A string that may contain regex operators.
+ * @return {string} A regex that matches on the input.
+ */
+export const escapeRegex = (str: string) =>
```

Comment:
[nit] Could include the return type
```suggestion
export const escapeRegex = (str: string): string =>
```

Reply:
![image](https://user-images.githubusercontent.com/4605522/150558860-fd4581e1-875c-4c8d-9af3-99e7e02746aa.png)
It gets inferred by typescript :)